src/containers/pages/InterventionPlan/UpdatePlan/index.tsx

```diff
@@ -93,6 +94,14 @@ const UpdatePlan = (props: RouteComponentProps<RouteParams> & UpdatePlanProps) =
   const planFormProps = {
     ...propsForUpdatingPlans,
     ...(plan && { initialValues: getPlanFormValues(plan) }),
+    /** a renderProp prop. this tells the planForm; I will give you a component that knows of the plan you are displaying,
+     * the component will get jurisdictions associated with that plan and render them as links, what you(planForm)
+     * will do, is provide a child prop(a renderProp) that tells the mentioned component what other stuff you would wish
+     * displayed alongside the jurisdiction names links.
+     */
+    renderLocationNames: (
+      child?: (locationName: string, locationId: string, index: number) => JSX.Element
+    ) => <ConnectedPlanLocationNames child={child} plan={plan} />,
```

Comment:
Should we move this typing to it's own line so it can be exported and used everywhere it needs to be defined?

Reply:
yeap.